guidelines.rst

```diff
@@ -154,6 +154,134 @@ The more stable the tier is, the stricter the rules for it are:
     - https://github.com/capi-workgroup/api-evolution/issues/42 (for limited API)
 
 
+One header
+==========
+
+All public API should be available after including :file:`Python.h`.
+
+To allow selecting an alternate API, such as a subset,
+use feature flags/macros that users define before including the header
+(for example ``Py_LIMITED_API``).
+Adding such a feature macro needs approval from the C API Working Group.
+
+.. note::
+
+    For background and discussions, see:
+
+    - https://github.com/capi-workgroup/api-evolution/issues/34
```

Comment:
Don't these need to be 3 space indented to match the `.. note`? (I'm not sure if that's a real ReST thing or not, but I learned to do it that way at some point, which I assume means a compiler shouted at me)